frontend/public/components/utils/units.js

```diff
@@ -382,3 +382,8 @@ export const convertToBaseValue = (value) => {
   // Unrecognized unit.
   return null;
 };
+
+export const secondsToNanoSeconds = (value) => {
+  const val = Number(value);
+  return Number.isFinite(val) ? val * 1000 ** 3 : 0;
```

Comment:
This is the [exponentiation operator](https://github.com/tc39/proposal-exponentiation-operator) which is semantically equivalent to `Math.pow`.

Nit: I'd put the expression `1000 ** 3` in braces to improve readability.

Reply:
eslint has a different opinion so I cannot add braces :)

Rebased

Reply:
Oh, ESLint :smiley